Client/src/ArmoniK.DevelopmentKit.Client.Common/Submitter/BaseClientSubmitter.cs

```diff
@@ -99,6 +99,12 @@ public class BaseClientSubmitter<T>
   [CanBeNull]
   protected ILogger<T> Logger { get; set; }
 
+
+  /// <summary>
+  ///   The number of chunk to split the payloadsWithDependencies
+  /// </summary>
+  public int ChunkSize { get; set; } = 5000;
```

Comment:
`get; init;` ?

Not sure what to expect if changed dynamically...

Reply:
Issue to push Init for all version of language. Instead private the Property to BaseClientSUbmitter and set by constructor